components/dash-core-components/src/fragments/RangeSlider.react.js

```diff
@@ -14,7 +14,12 @@ export default class RangeSlider extends Component {
             ? createSliderWithTooltip(Range)
             : Range;
         this._computeStyle = computeSliderStyle();
-        this.state = {value: props.value};
+
+        const {min, max, value, setProps} = props;
+        this.state = {value};
+        if (!value) {
+            setProps({value: calcValue(min, max, value)});
```

Comment:
`setProps` is going to fire a Dash callback. I don't think that's what we want here.